alvr/server_openvr/src/props.rs

```diff
@@ -577,6 +505,124 @@ pub extern "C" fn set_device_openvr_props(device_id: u64) {
                 ControllersEmulationMode::Custom { .. } => {}
             }
 
+            // Match for icons, to avoid duplicating across every Meta headset
+            // and also to override for the hand tracker controllers specifically
```

Comment:
I don't think the approach is good. you can maybe make a function (even a nested function) and call it multiple times. Index controllers should have their own icons.

Reply:
tbh I'm not sure why Index controllers don't have icons specified. Like, they actually do show up fine, but they weren't specified before either.

Reply:
Noone set them. This file is just full of random contributions

Reply:
You probably have seen this issue #262

Reply:
oh right, I can throw some more accurate details in after I refactor then